tests/test_integrity.py

```diff
@@ -12,6 +12,7 @@
 from astropy import units as u
 from astroquery.simbad import Simbad
 from astrodbkit2.utils import _name_formatter
+from astropy.constants import R_jup
```

Comment:
@dr-rodriguez , what do you think of this?  It makes the test pass but I'm not sure it's the best thing for us to do.

Reply:
Where is this used? From the changed lines the only place I see R_jup is as a string, not as a variable.

Reply:
It's as a string in the ModeledParameters table test data but is then tested as a unit. It passes 
```python
try:
     assert u.Unit(unit, parse_strict='raise')
```

But then, in the tests, we have to explicitly import which constants we're using as "units".  I think this is ok as long as we indicate in the docs.

Reply:
Interesting behavior on astropy.unit's part. I guess that's OK, but we may want to import a few more constants (M_Jup at least).

Reply:
It turns out that M_jup and R_jup are resolvable through from astropy import units as u and no addition imports are needed